ops/terraform/env/test/stateless/main.tf

```diff
@@ -22,4 +22,6 @@ module "stateless" {
   ssh_key_name    = var.ssh_key_name
   git_branch_name = var.git_branch_name
   git_commit_id   = var.git_commit_id
+
+  mpm_rda_cidr_block = "10.234.232.0/21"
```

Comment:
Ideally, we would store this in our vars file to prevent hard-coding and exposing inside network info to the public. Low risk, but it's something we are trying to steer away from.

Reply:
@keithdadkins can you link me to the vars file that would ideally hold this value?